WordPress/Classes/ViewRelated/Aztec/Renderers/SpecialTagAttachmentRenderer.swift

```diff
@@ -3,9 +3,9 @@ import UIKit
 import Aztec
 
 
-// MARK: - MoreAttachmentRenderer: This render is expected to only work with `<!--more-->` comments!
+// MARK: - SpecialTagAttachmentRenderer. This render aims rendering WordPress specific tags.
 //
-final class MoreAttachmentRenderer {
+final class SpecialTagAttachmentRenderer {
```

Comment:
Not a big fan of this name: `SpecialTagAttachmentRenderer` what about `CustomTagAttachmentRenderer` or `WordPressCommentAttachmentRenderer`.

Reply:
`WordPressCommentAttachmentRenderer` definitely no.  We should discourage closely-tying Aztec's code to WP as much as possible.

Reply:
Is this not WordpPress specific? And on the main WP app only?

Reply:
My bad!  I mistook this for the Example app :)

I think `CustomCommentAttachmentRendered` is the most appropriate name (since it's rendering a custom comment, as opposed to a tag).

Reply:
@diegoreymendez @SergioEstevao we already renamed this... twice. Mind if we just go forward with this name? i'm personally not a fan of the `CustomComment` prefix.